product_set/tests/test_product_set.py

```diff
@@ -10,9 +10,6 @@ class TestProductSet(common.TransactionCase):
     @classmethod
     def setUpClass(cls):
         super().setUpClass()
-        cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))
```

Comment:
@NICO-SOLUTIONS Why removing those lines ?

Reply:
there is no dependency to sale module in manifest. This test will be done in sale_product_set. So i guess this i not necessary here?!